jsapp/js/components/table.es6

```diff
@@ -79,37 +79,42 @@ export class DataTable extends React.Component {
       });
       filterQuery += '}';
       dataInterface.getSubmissions(this.props.asset.uid, pageSize, page, sort, [], filterQuery, true).done((data) => {
-        if (data.count) {
-          this.setState({resultsTotal: data.count});
-        }
+        this.setState({resultsTotal: data.count});
       });
     } else {
       this.setState({resultsTotal: this.props.asset.deployment__submission_count});
     }
 
     dataInterface.getSubmissions(this.props.asset.uid, pageSize, page, sort, [], filterQuery).done((data) => {
-      if (data && data.length > 0) {
+      let results = data.results;
+
+      if (results && results.length > 0) {
         if (this.state.submissionPager == 'next') {
-          this.submissionModalProcessing(data[0]._id, data);
+          this.submissionModalProcessing(results[0]._id, results);
         }
         if (this.state.submissionPager == 'prev') {
-          this.submissionModalProcessing(data[data.length - 1]._id, data);
+          this.submissionModalProcessing(results[results.length - 1]._id, results);
         }
         this.setState({
           loading: false,
           selectedRows: {},
           selectAll: false,
-          tableData: data,
+          tableData: results,
           submissionPager: false
         });
-        this._prepColumns(data);
+        this._prepColumns(results);
       } else {
         if (filterQuery.length) {
           this.setState({
-            loading: false
+            loading: false,
+            selectedRows: {},
+            tableData: results
           });
           // TODO: debounce the queries and then enable this notification
-          alertify.warning(t('The query did not return any results.'));
+          // Block the warning if selectAll is true
+          if (!this.state.selectAll) {
+            alertify.warning(t('The query did not return any results.'));
+          }
```

Comment:
Don't we want to get rid of this `alertify` call and display an empty message when there are zero results?

Reply:
I'm not sure I understand, an empty message?

Reply:
Instead of this:

<img width="335" alt="Screenshot 2019-08-08 at 20 09 55" src="https://user-images.githubusercontent.com/2521888/62727017-d4949700-ba18-11e9-9d6e-15bf59d5d1f8.png">

Do this:

<img width="808" alt="Screenshot 2019-08-08 at 20 11 51" src="https://user-images.githubusercontent.com/2521888/62727027-da8a7800-ba18-11e9-9bd1-db320b52aa45.png">

Reply:
What do you think?

Reply:
Sounds good I'll do that

Reply:
Could we change the wording to `Your filters returned no submissions.`?

Reply:
will do!